### PR TITLE
Postgres: Replaced username/password fields with Credentials store

### PR DESCRIPTION
**Pull request by omercnet**
This is breaking change when upgrading an existing instance

fixes https://github.com/demisto/etc/issues/12430

### CODE REVIEW

**Comment by gshriki** [comment]
@omercnet - let's either add a new integration, or provide a new YML to the customer. I do not want to break BC

**Comment by dorsha** [comment]
I think there is a simple way to handle it. Let's see tomorrow.

**Comment by dorsha** [comment]
No need for new integration or something like that.

**Comment by gshriki** [comment]
even better!

**Comment by gshriki** [comment]
@omercnet - discussed with Dorsha. Need to create logic in the integration itself to look for the old parameters if the new ones are not there. this way the integration can be backward compatible.

**Comment by dorsha** [comment]
1. It can be mandatory 
2. The backward check is only for old users which already have saved instance of the integration with the user/password fields

File: Integrations/integration-PostgreSQL.yml [hunk 1]
```diff
@@ -44,17 +39,22 @@ script:
           conn.close()
 
     def pgsqlConnect():
-        connectionString = "dbname='" + demisto.params()['dbname'] +"' ";
-        connectionString +="user='" + demisto.params()['user'] +"' ";
-        connectionString +="host='" + demisto.params()['host'] +"' ";
-        connectionString +="port='" + str(demisto.params()['port']) +"' ";
-        connectionString +="password='" + demisto.params()['password'] +"'";
+        params = demisto.params()
+        username = params.get('username') or params.get('credentials')['identifier']
```
Comment (omercnet) [inline Integrations/integration-PostgreSQL.yml]: @dorsha something like this?

Comment (dorsha) [reply Integrations/integration-PostgreSQL.yml]: Yes, just need to verify it works (and understand the using problem)

**Comment by omercnet** [comment]
tested locally and it works
the "using" issue is solved when recreating the instance

I'm guessing it's related to the  upgrade error issue but I'm guessing it doesn't apply to content update, is there a way to test/simulate a content update ?

**Comment by dorsha** [comment]
Yes will send you the details soon

**Comment by anara123** [comment]
@dorsha @omercnet what is the status of the pr?

**Comment by omercnet** [comment]
Blocked by https://github.com/demisto/etc/issues/12504

**Comment by Itay4** [comment]
Closing and keeping branch alive for now, until https://github.com/demisto/etc/issues/12504 is resolved.

**Pull request closed without merging**